Source/Objects/movementobject.cpp

```diff
@@ -1402,11 +1402,24 @@ void MovementObject::CreateRiggedObject() {
     rigged_object_->char_id = GetID();
     palette.clear();
     rigged_object_->SetCharacterScriptGetter(character_script_getter);
+
     if (GetScriptParams()->HasParam("Character Scale")) {
         rigged_object_->SetCharScale(GetScriptParams()->ASGetFloat("Character Scale"));
+
+        float new_frust = GetScriptParams()->ASGetFloat("Character Scale") * 2;
```

Comment:
This `ASGetFloat` call is already being done two lines above. Ideally we'd get rid of all these per-frame calls into the angelscript context entirely because this sort of variable lookup is relatively slow (compared to using the value of a variable defined and used from C++). But since you didn't put that code there, you definitely don't have to fix it. But ideally you also should avoid multiplying it.

For now, I think it'd be best to put the result of `ASGetFloat` in a local variable and reuse the value here.

Reply:
Oh, I expanded the code above. For some reason I assumed this was in draw, due to "frust" being the name of the variable. This is actually in `CreateRiggedObject`, so it's less of a concern.

Still, generally a good practice to avoid copying code that calls into chains of accessors. Just do it once in the function, assign it to a local variable, and reuse the result.